test/e2e/e2e_test.go

```diff
@@ -591,6 +592,57 @@ func pollForResource(t *testing.T, resourceID string, timeout time.Duration, acc
 	return obj, err
 }
 
+func pollForUpdatedServerCertFromClient(t *testing.T, client *kubernetes.Clientset, service *v1.Service, updatedServerCert string) error {
+	podName := "client-pod"
+	metricsURL := fmt.Sprintf("%s.%s.svc:%d", service.Name, service.Namespace, service.Spec.Ports[0].Port)
+	defer client.CoreV1().Pods(service.Namespace).Delete(podName, &metav1.DeleteOptions{})
+	_, err := client.CoreV1().Pods(service.Namespace).Create(&v1.Pod{
+		TypeMeta: metav1.TypeMeta{},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      podName,
+			Namespace: service.Namespace,
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name:    "cert-checker",
+					Image:   "nicolaka/netshoot:latest",
```

Comment:
We don't have any policy that would disallow it. I'd suggest maybe adding an ImageStream to `openshift/release/core-services/addiitional-images/` so that we don't need to pull it

Reply:
@stevekuznetsov thanks for the reply. I don't see `openshift/release/core-services/addiitional-images/` directory in release repo. Am i missing something?

Reply:
I typo'd that, sorry -- it's at `core-services/supplemental-ci-images/` in the `release` repo.